api/v1beta1/ovncontroller_types.go

```diff
@@ -175,6 +175,7 @@ type OVSExternalIDs struct {
 	// +kubebuilder:default={}
 	OvnAvailabilityZones []string `json:"availability-zones,omitempty"`
 
+	// This field is deprecated and might be removed when a new api version is released
```

Comment:
What about changing this to?
        // DEPRECATED: To be removed in the next API version
        // Any value set to this field is ignored